core/src/main/java/com/web3auth/core/Web3Auth.kt

```diff
@@ -182,10 +182,10 @@ class Web3Auth(web3AuthOptions: Web3AuthOptions) {
                     this.authorizeSession().whenComplete { resp, error ->
                         if (error == null) {
                             web3AuthResponse = resp
-                            initializeCf.complete(null)
                         } else {
                             print(error)
```

Comment:
This is just informational.

Instead of `print(error)` you could do this:
```
Log.e(tag, error);
```

Then you can use logcat with adb.